src/main/webapp/views/HomePage.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+  <main class="navy-blue">
+    <h1 class="home-title">starfish</h1>
+    <h3 class="subtitle">Share and find the best course notes. Completely free.</h3>
+    <!-- TODO: Connect these searchresult events to the search page -->
+    <search-bar></search-bar>
+    <img id="laptop" src="assets/laptop-min.png">
+  </main>
+</template>
+
+<script>
+module.exports = {
+  components: {
+    'search-bar': httpVueLoader('/components/SearchBar.vue'),
+    'mock-search': httpVueLoader('/components/MockSearchButton.vue'),
+  },
+}
+</script>
+
+<style>
+body {
+  margin: auto 0;
+}
+
+.zone {
+  font-size: 2em;
+}
+
+/* COLORS */
+.navy-blue {
+  background-color: #004aad;
+}
+
+/* NAV */
+.main-nav {
+  display: flex;
```

Comment:
Keely was keeping these properties sorted, I'd sync up with her to see if its what you all want to continue doing.